examples/typescript/index.ts

```diff
@@ -0,0 +1,16 @@
+const my_name = "Junya" as const;
+
+let age: number;
+age = 22;
+
+function get_array<T>(...args: T[]): T[] {
+	return [...args];
+}
+
+console.log(get_array("hoge", "huga"));
```

Comment:
I think `foo` and `bar` are better because `hoge` and `fuga` are used only Japan.